plugins/unicodelib/unicodelib.cpp

```diff
@@ -718,6 +719,48 @@ unsigned unicodeEditDistanceV4(UnicodeString & left, UnicodeString & right, unsi
     return da[mask(leftLen-1)][rightLen-1];
 }
 
+UnicodeString excludeNthWord(RuleBasedBreakIterator& bi, UnicodeString const & source, unsigned n)
+{
+	UnicodeString processed;
```

Comment:
whitespace: This file contains tabs to indent - our standard is to use 4 spaces.